README.rst

```diff
@@ -161,6 +164,128 @@ from the machine where you run gswrap).
 
     os.stat("/home/user/storage/file1").st_mtime # 1540889799
 
+* Copy a sequence of tuples from any source to any destination
+
+.. code-block:: python
+
+    sequence = [
+                    # copy on google cloud storage
+                    ('gs://your-bucket/your-dir/file',
+                    'gs://your-bucket/backup-dir/file'),
+                    # download
+                    ('gs://your-bucket/your-dir/file',
+                    pathlib.Path('/home/user/storage/backup-file')),
+                    # upload
+                    (pathlib.Path('/home/user/storage/new-file'),
+                    'gs://your-bucket/your-dir/new-file'),
+                    # copy locally
+                    (pathlib.Path('/home/user/storage/file'),
+                    pathlib.Path('/home/user/storage/new-file'))
+                ]
+    client.cp_many_to_many(srcs_dsts=sequence, recursive=False,
+                            no_clobber=False, multithreaded=False,
+                            preserve_posix=False)
+
+* Remove blobs from google cloud storage
+
+.. code-block:: python
+
+    # existing blobs:
+    # gs://your-bucket/file
+    client.rm(url="gs://your-bucket/file", recursive=False, multithreaded=False)
+    # bucket is now empty
+
+    # existing blobs:
+    # gs://your-bucket/file1
+    # gs://your-bucket/your-dir/file2
+    # gs://your-bucket/your-dir/sub-dir/file3
+    client.rm(url="gs://your-bucket/your-dir", recursive=True,
+              multithreaded=False)
+    # remaining blobs:
+    # gs://your-bucket/file1
+
+* Read and write to blobs
+
+.. code-block:: python
+
+    client.write_text(url:"gs://your-bucket/file", text="Hello, I'm text",
+                     encoding='utf-8')
+    client.read_text(url:"gs://your-bucket/file", encoding='utf-8')
+    # Hello I'm text
+
+    client.write_bytes(url="gs://your-bucket/data",
+                        data="I'm important data".encode('utf-8'))
+
+    data = client.read_bytes(url="gs://your-bucket/data")
+    print(data.decode('utf-8')) # I'm important data
+
+* Get and copy stats of file or metadata of blob
+
+.. note::
+
+    Copy on remote or locally always copies stats resp. metadata.
```

Comment:
stats respective?

Reply:
Does "stats respectively metadata" work? Is it common sense to not use abbreviations in the readme to prevent misunderstandings?